examples/notebooks/beam-ml/mltransform_basic.ipynb

```diff
@@ -0,0 +1,733 @@
+      "metadata": {
+        "id": "bdabbc57-ec98-4113-b37e-61962f488d61"
+      },
+      "outputs": [],
+      "source": [
+        "# store artifacts generated by MLTransform.\n",
+        "# Each MLTransform instance requires an artifact location to be empty.\n",
+        "# We use this method to delete and refresh the artifact location for each example.\n",
+        "artifact_location = './my_artifacts'\n",
+        "def delete_artifact_location(artifact_location):\n",
+        "  import shutil\n",
+        "  import os\n",
+        "  if os.path.exists(artifact_location):\n",
+        "      shutil.rmtree(artifact_location)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "28b1719c-7287-4cec-870b-9fabc4c4a4ef",
+      "metadata": {
+        "id": "28b1719c-7287-4cec-870b-9fabc4c4a4ef"
+      },
+      "source": [
+        "## Compute and map the vocabulary\n",
+        "\n",
+        "\n",
+        "`ComputeAndApplyVocabulary` is a data processing transform that computes a unique vocabulary from a dataset and then maps each word or token to a distinct integer index. It facilitates transforming textual data into numerical representations for machine learning tasks.\n",
+        "\n",
+        "Let's use `ComputeAndApplyVocabulary` with `MLTransform`\n",
+        "\n",
+        "\n",
+        "\n",
+        "\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "id": "56d6d09a-8d34-444f-a1e4-a75624b36932",
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "56d6d09a-8d34-444f-a1e4-a75624b36932",
+        "outputId": "2eb99e87-fb23-498c-ed08-775befa3a823"
+      },
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stderr",
+          "text": [
+            "WARNING:apache_beam.options.pipeline_options:Discarding unparseable args: ['-f', '/root/.local/share/jupyter/runtime/kernel-eb509d9c-cd3a-4a27-ab40-19bbfa38a5ad.json']\n",
+            "WARNING:absl:You are passing instance dicts and DatasetMetadata to TFT which will not provide optimal performance. Consider following the TFT guide to upgrade to the TFXIO format (Apache Arrow RecordBatch).\n",
+            "WARNING:absl:You are passing instance dicts and DatasetMetadata to TFT which will not provide optimal performance. Consider following the TFT guide to upgrade to the TFXIO format (Apache Arrow RecordBatch).\n",
+            "WARNING:absl:You are outputting instance dicts from `TransformDataset` which will not provide optimal performance. Consider setting  `output_record_batches=True` to upgrade to the TFXIO format (Apache Arrow RecordBatch). Encoding functionality in this module works with both formats.\n",
+            "WARNING:apache_beam.options.pipeline_options:Discarding unparseable args: ['-f', '/root/.local/share/jupyter/runtime/kernel-eb509d9c-cd3a-4a27-ab40-19bbfa38a5ad.json']\n"
+          ]
+        },
```

Comment:
consider removing irrelevant warnings like this manually

Reply:
These warnings are part of the tensorflow_transform code. They will appear if the user runs it. Do we want to remove those? since we will mention about them in the trouble shooting section.

Reply:
Yeah, I would still remove them manually. They don't add anything to the notebook and are distracted in the GitHub rendered version

Reply:
I will remove them. Thanks